evap/staff/templates/staff_user_form.html

```diff
@@ -14,7 +14,22 @@
 
 {% block content %}
     {{ block.super }}
-    <h3>{% if form.instance.id %}{% trans 'Edit user' %}{% else %}{% trans 'Create user' %}{% endif %}</h3>
+
+    <div class="d-flex mb-3">
```

Comment:
There is more whitespace now, let's make it look like before again :)
```suggestion
    <div class="d-flex">
```